dtests/src/test/java/io/snappydata/hydra/northwind/northWind.bt

```diff
@@ -5,6 +5,7 @@ io/snappydata/hydra/northwind/nwReplicatedRowTablesTest.conf
   D=worker workerHosts=3 workerVMsPerHost=1 workerThreadsPerVM=1
   dataFilesLocation="$GEMFIRE/../../../tests/common/src/main/resources/northwind/"
   tableType="ReplicatedRow"
+  fullResultSetValidation = true
```

Comment:
why this configurable ? We should always validate full resultset, so it should be always true.